### ISSUE TITLE
[Package Request] - Python3.8 to 3.11, autossh, iftop, R-4.2.2, Rstudio

### ISSUE THREAD

**Issue by RaresMiliton** [amazonlinux/amazon-linux-2023#281]
**What package is missing from Amazon Linux 2022? Please describe and include package name.**
On al2022 for Python the only available versions were 3.9 and 3.10. Now on al2023 only 3.9 is available, so 3.8, 3.10 and 3.11 are missing.
Other packages that are missing: autossh, iftop and Rstudio

**Is this an update to existing package or new package request?**
For now R version 4.1.3 is available, so 4.2.2 is the update request
For the rest it is just new package request

**Is this package available in Amazon Linux 2?**
iftop and autossh were previously available on AL2, and also Python3.8 through amazon-linux-extras

**Number of users impacted**
My organization, in order to migrate to al2023

**Comment by stewartsmith** [comment]
With the 2023.0.20230315 release of Amazon Linux 2023, which is now GA, we have `python3.11` packaged along with a `python3.11-pip` package. We are not currently looking to also package Python 3.8 or 3.10 on Amazon Linux 2023. Is there something specific about these versions which you are looking for?

**Comment by RaresMiliton** [comment]
We are needing at least Python3.10 as our organization is using it, and it was already available with AL2022

**Comment by stewartsmith** [comment]
Python 3.11 is available in AL2023 at GA. Is the older 3.10 needed, or is 3.11 adequate?

**Comment by RaresMiliton** [comment]
Yes, 3.10 is still needed, along with 3.8 for packages compatibility, but if 3.8 is out of question, we would still require at least 3.10

**Comment by CharlesPerrotMinotHCHB** [comment]
Bumping this up. To support his, we currently have to compile python 3.10 on AL2023, which isn't ideal.

**Comment by nkadel** [comment]
Packages which require python 3.11 include ansible-core and the mislabeled "ansible" package, which should be labeled "ansible_collections". Maintaining multiple python versions is quite painful. As an old multi-platform software builder, may I encourage you to update your underlying software to compatibility with python 3.11 ?

**Comment by stewartsmith** [comment]
Python 3.8 is coming up on EoL in October 2024 as per https://devguide.python.org/versions/ thus it won't be included in AL2023.

**Comment by sincerelysaucy** [comment]
I see some of python3.11 available for Amazon Linux 2023. The list of RPMs is not very large, and some packages like ansible now require python 3.12, which is not yet available.
